modules/api_management_api/outputs.tf

```diff
@@ -9,3 +9,7 @@ output "api_management_api_diagnostic" {
 output "application_id" {
```

Comment:
I think these outputs also need to be removed?

Reply:
Application is still made in api_management_api, only the role assignment itself is in a different module